extensions/wikia/SpecialPageViews/SpecialPageViewsController.class.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+class SpecialPageViewsController extends WikiaSpecialPageController {
+
+	const SPECIALPAGE_NAME = 'PageViews';
+
+	private $startDate, $endDate;
+
+	function __construct() {
+		parent::__construct( self::SPECIALPAGE_NAME );
+		$this->wg->Out->addModules( 'ext.SpecialPageViews' );
+	}
+
+	public function execute() {
+		wfProfileIn( __METHOD__ );
+		$this->wg->Out->setPageTitle( wfMessage( 'admindashboard-title' )->escaped() );
+
+		if ( !$this->wg->User->isAllowed( 'pageviews' ) ) {
```

Comment:
In general, prefer `$this->getUser()` over the global alias on special pages (and other context sources).